CHANGELOG.md

```diff
@@ -4,6 +4,32 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v13.3.0](https://github.com/voxpupuli/puppet-grafana/tree/v13.3.0) (2024-06-14)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-grafana/compare/v13.2.0...v13.3.0)
+
+**Breaking changes:**
+
+- Drop support for EoL Grafana 8 & 9 [\#361](https://github.com/voxpupuli/puppet-grafana/pull/361) ([bastelfreak](https://github.com/bastelfreak))
```

Comment:
Because of the breaking changes the next release has to be a major one.

Reply:
should be now